build.gradle.kts

```diff
@@ -1,44 +1,80 @@
 plugins {
-    kotlin("jvm") version "1.4.31"
-    kotlin("kapt") version "1.4.31"
-    kotlin("plugin.allopen") version "1.4.31"
-    id("io.micronaut.application") version "1.4.0"
+    id("org.jetbrains.kotlin.jvm") version "1.6.10"
+    id("org.jetbrains.kotlin.kapt") version "1.6.10"
+    id("org.jetbrains.kotlin.plugin.allopen") version "1.6.10"
+    id("com.github.johnrengelman.shadow") version "7.1.1"
```

Comment:
Why do we need the shadow plugin?

Reply:
I guess it's not necessary, it comes by default with micronaut, and the graalvm plugin by default uses the fat jar instead of the individual jars: https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#long_classpath_and_fat_jar_support

Using the shadow jar yourself may not actually make a difference, I am not sure if it uses that jar, or if it'll automatically use it's own fat-jarring. That may just be included with the micronaut thing.

Reading more about it, I don't think it's necessary. I'm taking it out. It looks like graalvm will do it's own fat jar stuff not using the shadow jar thing. Yeah, digging through the source code, it's just a way to deal with too-long classpaths in windows.